cmd/web.go

```diff
@@ -91,6 +111,9 @@ func runWeb(ctx *cli.Context) error {
 	case setting.HTTP:
 		err = runHTTP(listenAddr, context2.ClearHandler(m))
 	case setting.HTTPS:
+		if setting.RedirectPort80 {
+			go runHTTPRedirector()
```

Comment:
Can you elaborate on your concern?  If the redirector fails to start it will stop the server immediately with the call to log.Fatal right?